README.md

```diff
@@ -3,39 +3,32 @@
 [![Build Status](https://travis-ci.org/veres-one/did-veres-one.svg?branch=master&style=flat-square)](https://travis-ci.org/veres-one/did-veres-one)
 [![NPM Version](https://img.shields.io/npm/v/did-veres-one.svg?style=flat-square)](https://npm.im/did-veres-one)
 
+## Background
+
 This library provides support classes for creating and processing
 Decentralized Identifiers for [Veres One](https://veres.one/). This library
 enables a developer to:
 
 * Create a Veres One DID
 * Generate Veres One cryptographic proofs
 
-```
-npm install did-veres-one
-```
+### Compatibility
 
-```js
-const v1 = require('did-veres-one');
-
-// See Configuration below for list of options
-const options = {mode: 'dev', httpsAgent, hostname: 'localhost:12345'};
-const veresDriver = v1.driver(options);
-```
+* **`^v13.0.0`** - Compatible with the current Capybara testnet.
+* `^v14.0.0` - bleeding edge, not compatible with testnet.
```

Comment:
The language here seems a bit off and not forward looking on how this might be used going forward.  After discussion with @dmitrizagidulin we think a network-version mapping table might work better:
```suggestion
| Network | Client library version |
| :--- | :--- |
| Capybara Testnet | ^v13.0.0 |
| Local ledger dev/testing | ^v14.0.0 |
```

Reply:
@davidlehn thanks!!